index.html

```diff
@@ -185,6 +190,7 @@ <h3>Test repositories</h3>
         <li><a href="https://github.com/w3c/IMSC-1.1_Image_TestContent/pulse">Test contents for IMSC 1.1 Image</a></li>
         <li><a href="https://github.com/w3c/ttml1-tests/pulse">TTML1 tests</a></li>
         <li><a href="https://github.com/w3c/ttml2-tests/pulse">TTML2 tests</a></li>
+        <li><a href="https://github.com/w3c/ttml3-tests/pulse">TTML3 tests</a></li>
```

Comment:
I would remove this - I don't believe any work is happening on TTML3 tests at the moment.